catalog/app/embed/File.js

```diff
@@ -114,7 +114,7 @@ function VersionInfo({ bucket, path, version }) {
                   <M.ListItemText
                     primary={
                       <span>
-                        <FormattedRelative value={v.lastModified} />
+                        {dateUtils.formatRelative(v.lastModified)}
```

Comment:
using format function directly here is a little too naive, bc it doesnt handle memoization and updates like the [original component](https://github.com/formatjs/formatjs/blob/main/packages/react-intl/src/components/relative.tsx) did, so we probably should create a component that schedules updates and optionally memoizes the result